src/tabs/tabs.component.ts

```diff
@@ -65,6 +66,12 @@ export class Tabs implements AfterContentInit {
 	 * @memberof Tabs
 	 */
 	@Input() cacheActive = false;
+	/**
+	 * Set to 'true' to have tabs automatically activated and have their content displayed when they recieve focus.
+	 * @memberof Tabs
+	 */
+	@Input() automaticActivation = false;
```

Comment:
`automaticActivation` is a bit wordy ... either `activation: "auto" | "manual"` or `followFocus: boolean` may be better ... but that's kinda nit-picky.

for #254 it should default to `true`/`"auto"` as well